packages/core/src/app/checkout/CheckoutApp.tsx

```diff
@@ -56,10 +57,7 @@ const CheckoutApp = (props: CheckoutAppProps): ReactElement => {
             <LocaleProvider checkoutService={checkoutService}>
                 <CheckoutProvider checkoutService={checkoutService} errorLogger={errorLogger}>
                     <AnalyticsProvider checkoutService={checkoutService}>
-                        <ExtensionProvider
-                            checkoutService={checkoutService}
-                            errorLogger={errorLogger}
-                        >
+                        <ExtensionProvider extensionService={extensionService}>
```

Comment:
To decouple `contexts` from the `checkout-extension` package, we now inject `extensionService` into `ExtensionProvider` instead of creating it internally. 

Previously, it was instantiated within `ExtensionProvider` using `checkoutService` and `errorLogger`.